draft-irtf-cfrg-concrete-hybrid-kems.md

```diff
@@ -133,6 +149,119 @@ so that implementors can verify the correctness of their implementations.
 
 We make extensive use of the terminology in {{HYBRID-KEMS}}.
 
+# Designing a Concrete Instantiation
+
+A hybrid KEM is typically assembled using a choice of constituents that meets
+the needs of a certain application.  Considerations include:
+
+* The security level required by the application.
+* The application's assessment of the security of constituent groups / KEMs.
+* Compliance requirements that mandate the use of certain algorithms.
+* The availability of algorithm implementations in deployment environments.
+* Application design patterns, e.g., whether the same key is used many times.
+* The application performance impact of factors such as computation time or
+  key/ciphertext size.
+
+In {{HYBRID-KEMS}}, three frameworks are provided for constructing hybrid KEMs.
+Given the specific trade-offs that these frameworks present, there is typically
+one framework that will be optimal for a given application situation.
+Essentially, the GHP framework should be considered the default, with the QSF
+and PRE frameworks used in cases where applications are suited to their
+trade-offs:
+
+* The QSF framework requires the least computation, but imposes additional
+  security requirements on the constituents.  If the constituents meet the
+  requirements, then a QSF-based hybrid KEM will perform better than the other
+  two frameworks.
+
+* The PRE framework is faster than GHP in the cases where (1) encapsulation keys
+  are reused multiple times, and (2) encapsulation keys are large, but slower
+  when these conditions are not true.  So PRE should be preferred in situations
```

Comment:
Clarification on punctuation: "The PRE framework is faster than GHP in the cases where (1) encapsulation keys
  are reused multiple times and (2) encapsulation keys are large; however it is slower
  when these conditions are not true [XXXX]." 
Citation needed.